Sources/Quick/QuickSpec.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+
+// NOTE: This file is not intended to be included in the Xcode project. It
+//       is picked up by the Swift Package Manager during its build process.
+
+public class QuickSpec: XCTestCase, XCTestCaseProvider {
+    public func spec() {}
+
+    public required init() {}
+
+    public var allTests : [(String, () -> Void)] {
```

Comment:
I'm targeting the latest (January 11th) developer snapshot right now, which doesn't yet include that (breaking) change, but it will be necessary any day now.

Re: versioning, I'm not convinced that it makes sense to put too much effort into versioning until there's some kind of formal release (alpha, beta, whatever) that includes `corelibs` and `swiftpm`, which probably won't happen before the summer at least. In the meantime, anybody working with these tools should be aware that they are a moving target and expect breakage at every turn. To me, that is the strongest argument against merging this yet.